### DISCUSSION TITLE
OpenHAB Growatt Integration

### DISCUSSION THREAD

**Discussion by andrewfg** [johanmeijer/grott#483] [Show and tell]
You may be interested that I have integrated Growatt Inverter things into the OpenHAB eco system, by means of Grott.

Many Thanks!

- https://www.openhab.org/
- https://community.openhab.org/t/growatt-solar-inverter-binding/150600

**Comment by johanmeijer** [comment]
Can I refer to this in the documentation?

**Reply by andrewfg** [reply]
Sure. :)

**Reply by johanmeijer** [reply]
Thanks

**Reply by andrewfg** [reply]
@johanmeijer just to let you know that my PR for the openHAB Growatt binding, based on Grott, has been officially approved and merged into the openHAB main distribution. Many thanks for your kind work.

<img width="1133" alt="image" src="https://github.com/johanmeijer/grott/assets/893994/4ed14631-db07-4729-9fb7-a214c567fb7b">

<img width="1133" alt="image" src="https://github.com/johanmeijer/grott/assets/893994/7a1c4a16-9c1c-4855-a2e7-90596bb2f275">

**Comment by johanmeijer** [comment]
Wow that is nice! Great work! 

I think this is the link to the description: https://community.openhab.org/t/growatt-solar-inverter-binding-4-0-0-4-2-0/150600

**Reply by andrewfg** [reply]
> I think this is the link to the description

Yes. However that link is for the so called 'Market Place' version, and once the 'Official' version comes out (usually within 30 days), that  link may change. However I will send you the new link when it is released.

**Reply by andrewfg** [reply]
https://next.openhab.org/addons/bindings/growatt/
